common/persistence/sql/sqlplugin/sqlite/cluster_metadata.go

```diff
@@ -61,7 +61,7 @@ VALUES(?, ?, ?, ?, ?, ?, ?, ?) `
 
 	templatePruneStaleClusterMembership = `DELETE FROM
 cluster_membership 
-WHERE membership_partition = ? AND record_expiry < ? LIMIT ?`
+WHERE membership_partition = ? AND record_expiry < ?`
```

Comment:
I think we should remove LIMIT for all persistence, unless someone could explain me why we need it.

Reply:
I agree